tests/test_kafka.py

```diff
@@ -0,0 +1,32 @@
+from kafka import KafkaConsumer, KafkaProducer
+
+from testcontainers.kafka import KafkaContainer
+
+
+def test_kafka_producer_consumer():
+    with KafkaContainer() as container:
+        produce_and_consume_kafka_message(container)
+
+
+def test_kafka_producer_consumer_custom_port():
+    with KafkaContainer(port_to_expose=9888) as container:
+        assert container.port_to_expose == 9888
+        produce_and_consume_kafka_message(container)
+
+
+def produce_and_consume_kafka_message(container):
+    topic = 'test-topic'
+    bootstrap_server = container.get_bootstrap_server()
+
+    producer = KafkaProducer(bootstrap_servers=[bootstrap_server])
+    producer.send(topic, b"verification message")
+    producer.flush()
+    producer.close()
+
+    consumer = KafkaConsumer(topic, bootstrap_servers=[bootstrap_server])
+    consumer.partitions_for_topic(topic)
+    consumer.seek_to_beginning()
+
+    for msg in consumer:
```

Comment:
This test passes if there are no messages. I would suggest to assert that it's non-empty in addition or an assertion like "assert [m for m in consumer] == [expected_msg]".

Reply:
Using the code you suggested makes my test to wait indefinitely and eventually fail after some time. I have decided to use offset to verify that there is one message present on the topic before checking the contents.

Reply:
I see your point. However, the test as it is may still pass with the messages in the consumer being `[]` because the assertion within the `for` loop would never be executed. That's not ideal.

Alternatively, you could consider to use `poll` (see the [docs](https://kafka-python.readthedocs.io/en/master/apidoc/KafkaConsumer.html#kafka.KafkaConsumer.poll).

The `break` statement isn't needed.